src/jit/target.h

```diff
@@ -664,7 +664,11 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
   #define FEATURE_EH_FUNCLETS      1
   #define FEATURE_EH_CALLFINALLY_THUNKS 1  // Generate call-to-finally code in "thunks" in the enclosing EH region, protected by "cloned finally" clauses.
   #define FEATURE_STACK_FP_X87     0 
+#ifndef UNIX_AMD64_ABI
```

Comment:
It is a way to control functionality on Windows and System V ystems independently. I expect it to be needed.